Volume/Supporting/AppDelegate.swift

```diff
@@ -15,6 +16,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     private let notificationIntervalKey = "notificationIntervalKey"
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+        // Add SVG Support
+        SDImageCodersManager.shared.addCoder(SDImageSVGCoder.shared)
```

Comment:
How does adding a line here enable SVG rendering on other screens? Just curious.

Reply:
We were using the SDWebImage framework for all the screens, which does not support vector images by default. By registering this additional coder on launch, we can extend the WebImage support. 
https://github.com/SDWebImage/SDWebImageSVGCoder